cms/models.py

```diff
@@ -42,6 +42,12 @@ def can_create_at(cls, parent):
             and parent.get_children().type(cls).count() == 0
         )
 
+    def save(self, *args, **kwargs):
+        # autogenerate a unique slug so we don't hit a ValidationError
```

Comment:
as We moved the `save` method here so can you please remove from other child classes too ? If not required there now.

Reply:
`save` implementations seems me changed [here](https://github.com/mitodl/mitxpro/pull/225/files#diff-fe11e772bd1799d62172ed5b517988ebR105). Can you please confirm ?

Reply:
Will update shortly.